src/betterproto/__init__.py

```diff
@@ -780,20 +822,46 @@ def parse(self: T, data: bytes) -> T:
     # For compatibility with other libraries.
     @classmethod
     def FromString(cls: Type[T], data: bytes) -> T:
+        """
+        Parse the binary encoded Protobuf into this message instance. This
+        returns the instance itself and is therefore assignable and chainable.
+
+        .. note::
+            This is a method for compatibility with other libraries,
+            you should really use :meth:`parse`.
+
+
+        Parameters
+        -----------
+        data: :class:`bytes`
+            The data to parse the protobuf from.
+
+        Returns
+        --------
+        :class:`Message`
+        """
         return cls().parse(data)
 
     def to_dict(
         self, casing: Casing = Casing.CAMEL, include_default_values: bool = False
     ) -> Dict[str, Any]:
         """
-        Returns a dict representation of this message instance which can be
-        used to serialize to e.g. JSON. Defaults to camel casing for
-        compatibility but can be set to other modes.
-
-        `include_default_values` can be set to `True` to include default
-        values of fields. E.g. an `int32` type field with `0` value will
-        not be in returned dict if `include_default_values` is set to
-        `False`.
+        Returns a dict representation of this message instance which can be used to
```

Comment:
```suggestion
        Returns a JSON serializable dict representation of this object
```

note: I'm note sure referring to "this message instance" is the most technically correct, since the object will actually be an instance of a subclass of Message.